WebApplication1/WebApplication1/Controllers/DrinkController.cs

```diff
@@ -21,7 +21,9 @@ public DrinkController(IDrinkRepostory drinkRepostory, ICategoryRepostory catego
             this.categoryRepostory = categoryRepostory;
         }
 
-        public IActionResult List(string category)
+        // try to use async Task<IActionResult> coz it's better performance on production
+        // and will make ur api can proses more requests
+        public IActionResult List(string category)// ToDo you need to change this to accept the enum instead coz developer can make a typo which will break the work flow
```

Comment:
## using async Task<IActionResult>  will make the APIs accept more 
https://stackoverflow.com/a/41959672
but when you use the async try to go async all the way down